docs/developers/contributing.md

```diff
@@ -215,23 +213,6 @@ git push -u origin your-branch-name
 You can then make a
 [pull-request](https://guides.github.com/activities/forking/#making-a-pull-request) to `napari`'s `main` branch.
 
-## Building the docs
```

Comment:
removed this section because covers information already discussed in detail in Contributing Documentation.